examples/blueprint-construct/index.ts

```diff
@@ -171,6 +171,13 @@ export default class BlueprintConstruct {
             new blueprints.AwsBatchAddOn(),
             new blueprints.UpboundUniversalCrossplaneAddOn(),
             new blueprints.AwsForFluentBitAddOn(),
+            new blueprints.FluxCDAddOn({
```

Comment:
let's not bootstrap the repo, there are a couple of reasons:
1. The bootstrap repo contains helm charts for addons that are already provisioned through CDK which will cause a conflict.
2. Same case with Argo, resulted in inability to delete the stack. It may be a moot point, however, curious if you tested dropping the stack which had flux gitrepo bootstrapped?

Reply:
Done.